### DISCUSSION TITLE
New Win UI 3.0 Explorer (KB5030310) Gone, Why? - I Am Only Using The Windows 10 Start Menu.

### DISCUSSION THREAD

**Discussion by chiefisreal** [valinet/ExplorerPatcher#2257] [Q&A]
Title says it all but just using the windows 10 start menu which is great BTW the SEP moment 4 windows 11 update (KB5030310) revamped explorer with gallery and separable tabs is gone and now its using the old explorer with non separable tabs and no gallery. I don't know why considering I have it set to use stock windows 11 explorer so this shouldn't have been touched. Is their a way to fix this or will their be a update to bring this back soon?

Thanks!

**Comment by pyrates999** [comment]
That is a preview update: https://support.microsoft.com/en-au/topic/september-26-2023-kb5030310-os-build-22621-2361-preview-363ac1ae-6ea8-41b3-b3cc-22a2a5682faf

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.

**Comment by chiefisreal** [comment]
Hi

I am aware this is a preview update I don't know how that matters in this case.

Soo indirectly your saying that explorer patcher will eventually get patched and the new explorer will return. Thats all that I wanted to know. Still not sure how just using the windows 10 start menu tweak has effected explorer when I left that untouched but alrighty then

**Reply by pyrates999** [reply]
Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.
